docs/markdown/snippets/add_metrowerks_compiler.md

```diff
@@ -0,0 +1,5 @@
+## Added Metrowerks C/C++ toolchains
+
+Added support for the Metrowerks for Embedded ARM and Metrowerks for Embedded PowerPC toolchains (https://www.nxp.com/docs/en/reference-manual/CWMCUKINCMPREF.pdf).
+
+The implementation is somewhat experimental. It has been tested on a few projects and works fairly well, but may have issues
```

Comment:
Sentence is missing a period at the end.

Reply:
Whoopsie. Fixed.